_components/accordions.md

```diff
@@ -4,9 +4,153 @@ type: component
 title: Accordions
 ---
 
-<div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Accordions_UI_v1.png">
+<h3>Bordered Accordion</h3>
+
+<div class="code-sample">
+  <ul id="accordion-menu">
```

Comment:
Why are we using ids here? Using a class like `usa-accordion` would allow people to put multiple on the page, would lessen potential issues with specificity in the CSS code.

Let me know if you're OK with changing to IDs and I can update the code.